subsys/usb/class/usb_dfu.c

```diff
@@ -592,9 +592,12 @@ static int dfu_class_handle_req(struct usb_setup_packet *pSetup,
  *
  * @return  N/A.
  */
-static void dfu_status_cb(enum usb_dc_status_code status, const u8_t *param)
+static void dfu_status_cb(struct usb_cfg_data *cfg,
```

Comment:
how I see it is alternative to #14960, then please add fixes: #14882 to commit message

Reply:
That PR ( #14960) is a workaround. I am fine with it also before we merge the proper solution.